### PR TITLE
Fix/tca 389/remove unused method parameters

### PR DESCRIPTION
**Pull request by SashaTAO**
JIRA: https://oat-sa.atlassian.net/browse/TCA-389

Small refactoring of `importQtiItem` method. Renamed some variables for better understanding, removed unused parameters in method signature.

How to test:
- run QTI item import (UI + API)

related PR: https://github.com/oat-sa/extension-tao-testqti/pull/1769

### CODE REVIEW

File: model/qti/ImportService.php [hunk 1]
```diff
@@ -448,15 +445,12 @@ private function checkImportLockTime(float $startImportTime, string $itemId = ''
...
-        $folder,
+        $tmpFolder,
         Resource $qtiItemResource,
         $itemClass,
         array &$sharedFiles,
         array $dependencies = [],
         array $metadataValues = [],
-        array $metadataInjectors = [],
-        array $metadataGuardians = [],
-        array $metadataClassLookups = [],
```
[the remark this answers is not in the document]
Comment (SashaTAO) [reply model/qti/ImportService.php:459]: Done

File: model/qti/ImportService.php [hunk 2]
```diff
@@ -495,8 +486,7 @@ public function importQtiItem(
                             \common_Logger::i(
                                 'Resource "' . $resourceIdentifier . '" is already stored in the database and will not be imported.'
                             );
-                            $this->checkImportLockTime($startImportTime, $qtiItemResource->getIdentifier());
-                            $lock->release();
```
Comment (zagovorichev) [inline model/qti/ImportService.php:499]: Why have you deleted all releasing methods? We do need to release item on import finished, other processes that wait for this item will be blocked

File: model/qti/ImportService.php [hunk 3]
```diff
@@ -739,9 +725,11 @@ public function importQtiItem(
...
             $report = new common_report_Report(common_report_Report::TYPE_ERROR, __($e->getUserMessage()));
             $report->add($e);
+        } finally {
+            $this->checkImportLockTime($startImportTime, $qtiItemResource->getIdentifier());
+            $lock->release();
         }
-        $this->checkImportLockTime($startImportTime, $qtiItemResource->getIdentifier());
-        $lock->release();
+
         return $report;
```
Comment (zagovorichev) [inline model/qti/ImportService.php:733]: Report can be undefined, could you provided default value, please

**Comment by zagovorichev** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**